src/core/issues-notes.js

```diff
@@ -203,6 +193,36 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (issueSummaryElement) {
+    if (document.querySelectorAll(".issue").length) {
+      const heading = issueSummaryElement.querySelector("h2");
+      const paragraph = issueSummaryElement.querySelector("p");
+      if (heading && paragraph) {
+        issueSummaryElement.append(issueList);
+      } else if (!heading && paragraph) {
+        issueSummaryElement.removeChild(paragraph);
+        const issueSummary = hyperHTML`
+          <div><h2>${l10n.issue_summary}</h2>${paragraph}${issueList}</div>`;
+        issueSummaryElement.append(...issueSummary.childNodes);
+      } else {
+        const issueSummary = hyperHTML`
+          <div><h2>${l10n.issue_summary}</h2>${issueList}</div>`;
+        issueSummaryElement.append(...issueSummary.childNodes);
+      }
+    } else {
```

Comment:
I'm thinking that this is no longer an error. If the `issueList` doesn't have any children, then it should say: `<p>There are no issues listed in this specification.</p>`. The string should be localizable, but we can get to that.

Reply:
I figured out that function `handleIssues` is not called when there are no issues. Hence the function that i created is never called thus never providing me opportunity to show the error `<p>There are no issues listed in this specification.</p>`.

Check:

Reply:
If you want a different behaviour we will probably have to change this. I am not sure how to change this to meet ReSpec requirements.